CHANGELOG.md

```diff
@@ -1,18 +1,20 @@
 # 1.1.0 (2021-06-19)
 
+**Note:** We mark with (*) those PRs that adds features to the language.
```

Comment:
I'm not convinced this is how we should do things. The sign doesn't speak for itself, you need to be aware of the legend to understand it. And it's not very searchable.

If we want to highlight feature additions, I would use a tag like we're doing with `breaking-change` and `performance`.

However, feature additions should technically be easy to spot because their description should ideally start with `Add`. Looking at the marked changes, there's a high overlap with `Add`. Some are missing it, but could (and probably should) be rephrased. Then there's only additions to specs or docs which are not features, but also `Add` items. Maybe we should use a different indicator for that (for example `Improve`).

But this all feels like a discussion of its own and we might want to extract that to a different place?